src/overlaybd/tar/erofs/erofs_fs.cpp

```diff
@@ -21,11 +21,30 @@
 #include <dirent.h>
 #include <vector>
 
+struct erofs_fs_private {
+	struct erofs_sb_info sbi;
+	struct liberofs_file target_file;
+	unsigned long magic = LIBEROFS_MAGIC;
```

Comment:
what's the exact use of this fields?

Reply:
I mean what's the uses of `magic`?

Reply:
Just a double check for pointer conversion using reinterpret_cast.